numpy/core/src/npysort/selection.cpp

```diff
@@ -25,13 +25,113 @@
 #include <array>
 #include <cstdlib>
 #include <utility>
+#include "simd_qsort.hpp"
 
 #define NOT_USED NPY_UNUSED(unused)
 
+#if defined(_MSC_VER) || defined(__CYGWIN__)
+template<typename T>
+inline bool quickselect_dispatch(T* v, npy_intp num, npy_intp kth)
+{
+    return false;
+}
```

Comment:
Windows will never get the faster version?

Reply:
The quicksort patch ran into some problem with WIN32 builds and I never quite figured it out. Let me try running the partition patch through the CI, may be this one has better luck on windows.

Reply:
Looks like this one had no trouble. Windows will get the faster version too :) Might be worth checking if quicksort can be enabled, but will do it in a separate PR.

Reply:
It seems to work. Should we circle back to enabling AVX512 quicksort for windows?

Reply:
Yup, separate PR?

Reply:
Yes, sorry, separate PR would be good.

Reply:
AVX512 quicksort, argsort, partition and argpartition are all enabled on WIN32.